screens/login_screen/login_screen.gd

```diff
@@ -2,7 +2,7 @@ extends Control
 
 export (PackedScene) var next_scene
 
-var check = true # set to false to disable connecting to server
+var check = false # set to false to disable connecting to server
```

Comment:
lol just make sure to leave this as true

Reply:
always forget 👍